src/api/index.js

```diff
@@ -21,6 +21,36 @@ const METHOD = {
   POST: 'POST',
 };
 
+export const v4 = {
+  root: 'https://api.github.com/graphql',
+  call: async parameters => {
+    const response = await fetch(v4.root, parameters);
+    const json = JSON.parse(response._bodyText);
+
+    if (response.status >= 200 && response.status < 300) {
+      return json.data;
+    }
+    const error = new Error(json.message);
+
+    error.response = response;
+    throw error;
+  },
+  parameters: (accessToken, body = {}) => {
+    return {
+      method: METHOD.POST,
+      headers: {
+        Authorization: `token ${accessToken}`,
+      },
```

Comment:
Haven't tried GraphQL on my side yet. Are this header enough? Shouldn't we be passing a User-Agent ? (GitPoint/Android-3.2 for example).

I'd like to have it shared between GraphQL & Rest

Reply:
This is all that's necessary to use the endpoint, though I could add a user-agent header. See the example at https://developer.github.com/v4/guides/forming-calls/#communicating-with-graphql.

For trying the GraphQL API, I like https://developer.github.com/v4/explorer/, which lets you run queries as your GitHub user in the browser.